Makefile

```
@@ -1,7 +1,30 @@
-test-yes:
-	./tests/yes.sh
+# Export GO111MODULE=on to enable project to be built from within GOPATH/src
+export GO111MODULE=on
+GO_PACKAGES=$(shell go list ./... | grep -v vendor)
+.PHONY: lint \
+        deps-update \
+        vet
+vet:
+	go vet ${GO_PACKAGES}
```

```suggestion
	go vet ./...
```

Since `GO_PACKAGE` is only used in one place, I would remove the variable. 
According to `go help packages`, the `vendor` directory will excluded while `./...` is used:
```
Second, any slash-separated pattern element containing a wildcard never
participates in a match of the "vendor" element in the path of a vendored
package, so that ./... does not match packages in subdirectories of
./vendor or ./mycode/vendor, but ./vendor/... and ./mycode/vendor/... do.
```

---

I can update after I add the vendor dir and go.mod, go.sum files needed for for make to run.